chat-widget/package.json

```diff
@@ -91,14 +91,12 @@
     "compose-storybook": "start-storybook -c stories/.storybook -p 9009",
```

Comment:
Can we add a `lint` script to run `yarn eslint .`? It will be easier for folks to run linting locally